CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # legacysurvey Change Log
 
-## 6.0.0 (DR6, )
+## 6.0.0 (DR6, 2018-2-23)
```

Comment:
One more minor thing: two-digit month please, for consistency with the rest of the file.